cmd/exo/table/table.go

```diff
@@ -47,3 +47,15 @@ func (t *Table) Render() {
 		t.Table.Render()
 	}
 }
+
+// RemoveFrame remove all border and separator
+func (t *Table) RemoveFrame() {
+	t.SetBorder(false)
+	t.SetColumnSeparator("")
+	tablewriter.PadLeft("", "", 0)
+}
```

Comment:
you endup with that one:

https://golang.org/pkg/text/tabwriter/

Reply:
thx